fdbclient/include/fdbclient/BlobCipher.h

```diff
@@ -944,7 +944,7 @@ class DecryptBlobCipherAes256Ctr final : NonCopyable, public ReferenceCounted<De
 	EVP_CIPHER_CTX* ctx;
 	BlobCipherMetrics::UsageType usageType;
 	Reference<BlobCipherKey> textCipherKey;
-	Optional<Reference<BlobCipherKey>> headerCipherKey;
+	Optional<Reference<BlobCipherKey>> headerCipherKeyOpt;
```

Comment:
nit: no need to wrap `Reference` with `optional`; given `Reference` already can be `nullable`

Reply:
from older commit